.github/workflows/pr-verifier.yaml

```
@@ -0,0 +1,19 @@
+name: PR Verifier
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize]
+
+permissions:
+  checks: write
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    name: verify PR contents
+    steps:
+    - name: Verifier action
+      id: verifier
+      uses: kubernetes-sigs/kubebuilder-release-tools@e389278c2051863fb39b8de00159c8d7fac760ea # tag=v0.4.1
```

```suggestion
      uses: kubernetes-sigs/kubebuilder-release-tools@e389278c2051863fb39b8de00159c8d7fac760ea # v0.4.1
```
It should be withotu `tag=` part. Only single workflow in IPAM has `tag=`, which we should correct to be consistent.

---

fixed